pybossa/view/account.py

```diff
@@ -1149,8 +1149,8 @@ def add_metadata(name):
                                input_form=True,
                                can_update=can_update,
                                upref_mdata_enabled=bool(app_settings.upref_mdata),
-                               country_name_to_country_code=app_settings.upref_mdata.country_name_to_country_code(),
-                               country_code_to_country_name=app_settings.upref_mdata.country_code_to_country_name())
+                               country_name_to_country_code=app_settings.upref_mdata.country_name_to_country_code,
```

Comment:
Just confirming if this change was intentional?

Reply:
Yes, it is.